.travis.yml

```diff
@@ -1,17 +1,30 @@
-language: scala
-scala:
-- 2.11.2
-jdk:
-- oraclejdk8
-install: mvn install -P !build-extras -DskipTests=true -Dmaven.javadoc.skip=true -B
-  -V
-script: mvn test -P !build-extras -B
-cache:
-  directories:
-  - "~/.m2/repository"
-after_success:
-- "./cd/before-deploy.sh"
-- "./cd/deploy.sh"
+matrix:
+  include:
+    - language: scala
+      scala:
+      - 2.11.2
+      jdk:
+      - oraclejdk8
+      install: mvn install -P !build-extras -DskipTests=true -Dmaven.javadoc.skip=true -B
+        -V
+      script: mvn test -P !build-extras -B
+      cache:
+        directories:
+        - "~/.m2/repository"
+      after_success:
+      - "./cd/before-deploy.sh"
+      - "./cd/deploy.sh"
+    - language: "python"
+      python:
+        - 3.6
+      before_script: cd python
+      install:
+        - make
```

Comment:
this is being invoked from the root dir but the makefile is in `python/`.  You could do `make python/Makefile`